source/features/open-notification-in-new-tab.js

```diff
@@ -0,0 +1,16 @@
+import select from 'select-dom';
+import {registerShortcut} from './improve-shortcut-help';
+
+export default function () {
+	registerShortcut('notifications', 'shift o', 'Open Notification in New Tab');
+
+	document.addEventListener('keyup', event => {
+		const notification = select('.js-notification, .navigation-focus');
+		const isO = (event.key === 'O');
```

Comment:
This can just be inline in the `if` statement.

Reply:
Ha, definitely..